### ISSUE TITLE
Building SS plan while upgrading SS Lab

### ISSUE THREAD

**Issue by mangust404** [plainblack/Lacuna-Web-Client#175]
When SS Lab "A" building is upgrading (white timer is ticking) and building any of SS plan is complete (yellow timer is 00:00:00) then both timers disappears and building appears like upgraded with increased level.
Only re-entering the colony makes the white timer to appear correctly.

**Comment by 1vasari** [comment]
Yeah. There's all sorts of funk like that happening in the planet view. We'll get it sorted out sometime.
